src/tests/unit/auto/parse_meta_device_test.cpp

```diff
@@ -37,11 +37,13 @@ using ::testing::InvokeWithoutArgs;
 using Config = std::map<std::string, std::string>;
 using namespace MockMultiDevice;
 
-const char cpuFullDeviceName[] = "Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz";
+// const char cpuFullDeviceName[] = "Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz";
 const char igpuFullDeviceName[] = "Intel(R) Gen9 HD Graphics (iGPU)";
-// const char dgpuFullDeviceName[] = "Intel(R) Iris(R) Xe MAX Graphics (dGPU)";
-const char myriadFullDeviceName[] = "Intel Movidius Myriad X VPU";
-const char vpuxFullDeviceName[] = "";
+const char dgpuFullDeviceName[] = "Intel(R) Iris(R) Xe MAX Graphics (dGPU)";
+// const char myriadFullDeviceName[] = "Intel Movidius Myriad X VPU";
+// const char vpuxFullDeviceName[] = "";
+const std::vector<std::string>  availableDevs = {"CPU", "GPU.0", "GPU.1",
+    "MYRIAD.9.2-ma2480", "MYRIAD.9.1-ma2480", "VPUX"};
```

Comment:
ma2480 is a firmware name, it may change someday, why do we need that here?

Reply:
from the hello_query_device, the device name is "MYRIAD.9.2-ma2480", "MYRIAD.9.1-ma2480", so I use them as MYRIAD device name of mock unit test

Reply:
The device names here are used for mock unit test only. This mock test case will not be impacted by real HW plugin change.